package.json

```diff
@@ -1,34 +1,74 @@
 {
   "name": "aurelia-pal-nodejs",
-  "description": "node-js implementation of the aurelia pal-abstraction layer",
-  "author": "Meirion Hughes <crakinshot@yahoo.com>",
+  "version": "0.1.0",
```

Comment:
Ok, should we just keep use the `alpha` tag and release a `1.0.0-alpha.2.0.0` next? @MeirionHughes @EisenbergEffect